common/changes/@cadl-lang/compiler/HoverBranch_2022-08-15-20-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/compiler",
+      "comment": "implementation of hover feature ",
```

Comment:
Try to give this a name that is a bit more meaningful for the end user. This produce the changelog, someone should be able to look at that entry and understand what new feature was added.